projects/core/components/notification/notification.component.ts

```diff
@@ -19,7 +20,7 @@ import {TuiNotificationT} from '@taiga-ui/core/types';
 import {Observable} from 'rxjs';
 
 @Component({
-    selector: 'tui-notification',
+    selector: 'tui-notification, [tuiNotification]',
```

Comment:
let's add `:is(a, button)` so that it wouldn't suggest this selector for every tag.

Reply:
fixed